packages/installCatalog.sh

```diff
@@ -38,14 +38,14 @@ PACKAGE_HOME="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 export WSK_CONFIG_FILE= # override local property file to avoid namespace clashes
 
-# clone all Blueprints
-for bp in blueprint-hello-world blueprint-cloudant-trigger blueprint-messagehub-trigger
+# clone all Templates
+for bp in template-hello-world template-cloudant-trigger template-messagehub-trigger
```

Comment:
Assuming `bp` stands for blueprint. Maybe use an abbreviation for template instead (tml).